src/polyswarmtransaction/__main__.py

```diff
@@ -0,0 +1,28 @@
+import getpass
+import json
+
+import click
+import web3
+
+from .transaction import CustomTransaction, HexBytes
+
+
+@click.command()
+@click.argument('keyfile', type=click.File())
+@click.option('--payload', type=click.File(mode='rb'), default='-', show_default=True)
+@click.option('--password', '-p')
```

Comment:
Does getpass do the same thing? Is there an advantage of getpass over the builtin click functionatily?

Reply:
getpass is a bit better because it always rescue to tty even if stdin is not tty. This allows the payload to be stdin, something that eases the Scala integration.

Also, getpass is Python stdlib.